ldi-core/ldes-client/tree-node-supplier/src/main/java/ldes/client/treenodesupplier/repository/filebased/FileBasedMemberRepository.java

```diff
@@ -0,0 +1,57 @@
+package ldes.client.treenodesupplier.repository.filebased;
+
+import ldes.client.treenodesupplier.domain.entities.MemberRecord;
+import ldes.client.treenodesupplier.domain.valueobject.MemberStatus;
+import ldes.client.treenodesupplier.repository.MemberRepository;
+import ldes.client.treenodesupplier.repository.filebased.mapper.MemberRecordMapper;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class FileBasedMemberRepository implements MemberRepository {
+	public static final String UNPROCESSED_MEMBERS = "membersUnprocessed.txt";
+	public static final String PROCESSED_MEMBERS = "membersProcessed.txt";
```

Comment:
Did I miss something or do we write all processed members to this file with their model included? What will this file look like when I sync with an LDES with 1 mio members?

Reply:
Once, a member is processed its model is set to `null` so the file memberProcessed.txt only contains ids en statusses